src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,65 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+/**
+ * Ui class which handles interactions with the user
+ */
+public class Ui {
+    private static final String BORDER = "-------------------------------------";
```

Comment:
I like that the name was capitalized to follow coding standards for constants. 😄